client/pkg/caller/caller.go

```diff
@@ -28,11 +28,31 @@ type (
 	Component string
 )
 
+// nolint: exported
```

Comment:
Is this necessary?

Reply:
Yes, otherwise it would require a comment for each variable